docs/guide/basics/release-cycle.md

```diff
@@ -12,9 +12,35 @@ So assuming next version is 1.x the two-month cycle will look as following:
 - v1.x-RC.y - unstable version with cutting-edge features ready to test and feedback
 - v1.x.y - stable version of the software **ready for production use**.
 
-# How new features are merged
+## How new features are merged
 
 During RC features Pull Request with new features after feedback and acceptance are normally merged to `develop` branch. 
 After entering the Stabilization Phase we are tagging current develop branch, creating a `RC-x` (where `x` is a number of current version) branch from it and working on stabilization there. 
 During the stabilization phase new features are merged to develop branch and will be merged on next `RC` phase.
 
+## Release cycle flow
+
+### 1. Development phase
+
+The first phase of cycle we're mostly focusing on features and improvements. Branches in that phase should be created from actual `develop`, also PRs should be pointed to this branch. Changes merged to develop are available to test on https://test.storefrontcloud.io
+
+![Development phase](./assets/release-cycle-1.png)
+
+### 2. Release Candidate phase
+
+At some point, when milestone for next minor versions is completed, we're creating new branch from `develop` called `RC-x.y` (example: `rc-1.9`).
+After that new branch is tagged as first RC for version (example `v1.9.0-RC.1`). Then it's ready for testing by community.
+During tests, feedback and stabilisation there could be multiple RC releases on this branch. When improvement is made on this phase, then branch should be created from actual `rc-x.y` and should not contain features at this point - only improvements for current release.
+When release branch is updated, then should `develop` should be synchronized with it.
```

Comment:
I think the `develop` should be updated during (any) version release, same as in git-flow.

So `release/*` branch should be merged simultaneously to `master` and `develop`.

Reply:
i disagree with that. Release branch should be merged only to develop. To master should be merged only to change RC to stable version.
So:
- changes in release branch(stabilization fixes) -> merge to develop
- changes in master branch(hotfix) -> merge to release branch and develop branch

Reply:
The  idea is to have
- `master` should be always a one with stable, production ready code. That means only releases from stabilization phase and hotfixes are applied there
- `develop` is a working branch and should be always up to date with latest code no matter if we are on RC or STABILIZATION phase

Reply:
So how to release RC version? Tag RC releases on this temporary branch? I don't think it's a good idea.

I know that probably lots of people are just based on `master` branch and don't give a f about tags, but maybe they should start :P

Reply:
```suggestion
After merging a set of bugfixes and improvements into `release/*` branch, it needs to be tagged as the next RC version and merged into the `develop` branch to update it.
```

Reply:
So if we want to tag on this temporary branch, we should make it clear (also there is now "should" two times and in general this senstence is kinda weird :D )

```suggestion
After merging a set of bugfixes and improvements into `release/*` branch, it needs to be tagged as the next RC version and merged into the `develop` branch, to update it.
```